src/theory/quantifiers/cegqi/ceg_instantiator.h

```diff
@@ -186,6 +186,24 @@ enum CegInstPhase
 
 std::ostream& operator<<(std::ostream& os, CegInstPhase phase);
 
+/**
+ * The handled status of a sort/term/quantified formula, indicating whether
+ * counterexample-guided instantiation handles it.
+ */
+enum CegHandledStatus
```

Comment:
Clarify at a high level what is the difference between each of these states. What separates e.g. UNHANDLED from PARTIALLY_UNHANDLED?

Reply:
it depends on the context, I've added further documentation above isCbqiQuant.

Reply:
I think this would be more clear if you did not use "HANDLED" for "EXCLUSIVE" and "PARTIALLY_HANDLED" to "SHARED WITH OTHER INST TECHNIQUES." 

Also it's still not clear, looking only at the documentation, what UNHANDLED_UNCONDITIONAL means.